configure.ac

```diff
@@ -761,9 +778,9 @@ AC_ARG_WITH([lua], [AS_HELP_STRING([--with-lua], [build with lua support])],
 
 AS_IF([test "$with_lua" != no],[
   PKG_CHECK_MODULES([LUA],
-    [lua >= 5.1],
+    [lua >= 5.2],
```

Comment:
Why aren't we just bumping to Lua 5.3? That's what we _actually_ test with and our current compatibility base is with.

Reply:
Because 5.2 is the actually required minimum? See commit 6d4c68ba10b9713f3e599cf20c2455eb80e9bfe1

Reply:
Heh, it's funny how easy is to misread the subject line of that commit message when shown in a popup box after hovering over it. It says "drop support for Lua < 5.2" with a line break after the "<" symbol which is then easy to miss :)

Reply:
Hah, I thought we actually dropped Lua 5.2 support as well. Okay then...